template.yaml

```diff
@@ -1335,6 +1335,11 @@ Outputs:
     Description: BAV Custom Domain API Gateway ID
     Value: !Ref BAVFrontCustomDomain
   BAVIPVStubExecuteURL:
+    Condition: IsNotProdLikeEnvironment
+    Description: "BAV IPV Stub"
+    Value:
+      Fn::ImportValue: !Sub '${IPVStubStackName}-IPVStubPrettyApiUrl'
+  BAVIPVStubPrettyAPIURL:
```

Comment:
Is the diff showing strange or is the value for `BAVIPVStubExecuteURL` the pretty url and the value for `BAVIPVStubExecuteURL` the non pretty url?